service/src/main/java/de/kreggel/auth/service/registration/Userdata.java

```diff
@@ -0,0 +1,37 @@
+package de.kreggel.auth.service.registration;
+
+import java.io.Serializable;
+
+/**
+ * Created by oliverklette on 1/2/16.
+ */
+public class Userdata implements Serializable{
```

Comment:
!fix we should change the name. We will have multiple ways to register. Main: Facebook
Optional: Twitter, Github or direct username/password